api_app/playbooks_manager/serializers.py

```diff
@@ -38,6 +38,20 @@ class Meta:
     tags = TagSerializer(required=False, allow_empty=True, many=True)
     tlp = rfs.CharField(read_only=True)
     weight = rfs.IntegerField(read_only=True, required=False, allow_null=True)
+    is_deletable = rfs.SerializerMethodField()
+
+    def get_is_deletable(self, instance: PlaybookConfig):
+        # a playbook is deletable if is not a default one
+        if instance.owner:
+            # if is a custom playbook,
+            # it is deletable by the owner of the playbook or by an admin
+            if (
+                instance.owner != self.context["request"].user
+                and not self.context["request"].user.membership.is_admin
+            ):
+                return False
+            return True
+        return False
```

Comment:
Three things
- if the user is System administrator (not org admin), every playbook is deletable at the moment (he could do it in django admin anyway)
- the second `if` condition, write it in a positive direction, otherwise my brain will explode
- the second part of the second `if` condition is missing a condition: `AND` the user requesting playbook `must` be of the same organization of the owner of the playbook. Otherwise an organization owner can delete playbooks of other organizations